README.md

```diff
@@ -6,7 +6,7 @@
 
 ![license-badge](https://badgen.net/badge/license/GPLv3/blue)
 ![example](https://github.com/uab-cgds-worthey/rosalution/actions/workflows/python.yml/badge.svg)
-[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/uab-cgds-worthey/rosalution/blob/main/CODE_OF_CONDUCT.md)
```

Comment:
For non-templates, this can be relative URLS so that they are accurate to the state within the branch it is in.